docs/adapters.md

```diff
@@ -2,6 +2,43 @@
 title: "Playback adapters"
 ---
 
-# Playback adapters!
+# Playback adapters
 
-TO BE DONE
+## Basic concept
+
+There are lot of varios formats and concept of delivering video to end user.
+Some times you could be satisfied with [progressive download](https://en.wikipedia.org/wiki/Progressive_download) of your mp4 video.
+Main pros of that approach - crossbrowser support and almost no specific requirements for your backend.
+But when user experience is really important for you, you should conside to use [adaptive streaming](https://en.wikipedia.org/wiki/Adaptive_bitrate_streaming).
+Main pros - better user experience. Main cons - there is no crossbrowser support of main formats (HLS and MPEG-DASH) of adaptive streaming. But there is various libraries that add this support to browser. But they are heavyweigth, so we are adding them to our default bundle, but we give you ability to add them to Playable.
```

Comment:
A lot of "Main pros" and "But" phrases.

My poor attempt to improve that:
```
There are a lot of various formats and concept of delivering video to end user.
You could be satisfied with the progressive download of your mp4 video, which has cross-browser support and almost no specific requirements for your backend.
But if the user experience is really important for you, you should consider using adaptive streaming.
Pros - better user experience.
Cons - there is no cross-browser support of main formats (HLS and MPEG-DASH) of adaptive streaming.
There are various libraries that add this support to the browser.
We did not add them to our default bundle because they are heavyweight, but we give you the ability to use them.
```

Reply:
![image](https://user-images.githubusercontent.com/2445828/36540737-25c47e20-17e4-11e8-8837-cf6d36d82fc9.png)

"are NOT adding them"?